examples/experimental/i3s-picking/mesh-layer/mesh-layer.js

```diff
@@ -52,14 +54,16 @@ export default class MeshLayer extends SimpleMeshLayer {
   initializeState() {
     super.initializeState();
 
-    this.state.attributeManager.add({
-      pickingColors: {
-        type: GL.UNSIGNED_BYTE,
-        size: 3,
-        noAlloc: true,
-        update: this.calculatePickingColors
-      }
-    });
+    if (this.props.layerType === OBJECT_3D_LAYER_TYPE) {
```

Comment:
Can't we just check if the data on the tile is populated? Do we need to know the type? 

Better to avoid plumbing through such things if we can avoid it...